packages/react-scripts/config/webpackDevServer.config.js

```diff
@@ -72,10 +72,12 @@ module.exports = function(proxy, allowedHost) {
     // WebpackDevServer is noisy by default so we emit custom message instead
     // by listening to the compiler events with `compiler.plugin` calls above.
     quiet: true,
-    // Reportedly, this avoids CPU overload on some systems.
+    // Reportedly, ignoring node_modules avoids CPU overload on some systems.
     // https://github.com/facebookincubator/create-react-app/issues/293
+    // src/node_modules is not ignored to support absolute imports
+    // https://github.com/facebookincubator/create-react-app/issues/1065
     watchOptions: {
-      ignored: /node_modules/,
+      ignored: /^([^\/]|\/(?!src))*\/node_modules\//,
```

Comment:
Also, I believe this "breaks" for things like `src2`, as it checks for a prefix of `src`. Major edgecase.